engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/jobrunner/FlinkMiniClusterScenarioJobRunner.scala

```diff
@@ -79,11 +60,8 @@ class FlinkMiniClusterScenarioJobRunner(
 
   override def liveDataPreviewSupport: LiveDataPreviewSupport = {
```

Comment:
Why `liveDataPreviewSupport` is a part of `ScenarioJobRunner` if it is not related with it? I don't see any occurrencces of this method usage from the job runner.